concent_api/command_line_tool/message_handler.py

```diff
@@ -0,0 +1,105 @@
+import os
+import requests
+from golem_messages.shortcuts import dump
+from golem_messages.shortcuts import load
+from golem_messages.message.concents import ForceGetTaskResult
+from golem_messages.message.concents import ForceReportComputedTask
+from golem_messages.message.concents import ForceSubtaskResults
+from golem_messages.message import AckReportComputedTask
+
+from api_testing_common import print_golem_message
```

Comment:
Please keep alphabetical order.

Reply:
Done.

Reply:
Almost good, apart from:
```from golem_messages.message import AckReportComputedTask```
should be before 
```from golem_messages.message.concents import ForceGetTaskResult```
etc. If the import imports from 'higher' module it should be first

and 

```
from utils.helpers import sign_message
from utils.helpers import get_field_from_message
```
it should be opposite, please note that if imports are from same module we do alphabetic order on imported names.

Reply:
@MichalW-CP What about this one ?